pages/docs/how-to-guides/events.mdx

```diff
@@ -25,6 +25,10 @@ _If you need help on how to edit this file, please see the <Link href="/docs/how
 <ClipboardCopy>
 ```js
 {
+  "userStatus": "speaking",
+  "speakingTopicDetails": {
```

Comment:
is there any other fields inside `speakingTopicDetails`? if not just `topic` filed would be good instead of an object

Reply:
Maybe we can add other fields later, that's why I made it as an object.  Like `"description" : ""  etc.`